python/mor/reduction/reduceModel.py

```diff
@@ -262,7 +263,7 @@ def copyFileIntoAnother(self,fileToCopy,fileToPasteInto):
         except:
             raise
 
-    def copyAndCleanState(self,results,periodSaveGIE,stateFileName,gie=None):
+    def copyAndCleanState(self,results,periodSaveGIE,stateFileName,velocityFileName,gie=None):
```

Comment:
it will be better if this new argument was by default None and in this case we save nothing.
Maybe we can see also to have an argument allowing to either save state, or velocity or even both...

Reply:
I added an option to save both position and velocity snapshots, or save only position. It's an "intermediate" solution, as velocity vectors are always saved (=> write state file for velocity is always added in the scene).
But the velocity state file is only built if the option is set to true.